cmd/ingress-validator/ingress-validator.go

```diff
@@ -1,22 +1,23 @@
 package main
 
 import (
-	"fmt"
-	"os"
 	"crypto/tls"
 	"crypto/x509"
+	"fmt"
+	"os"
+	"strconv"
```

Comment:
my editor auto-formatted a bunch of stuff on save; nothing looks unreasonable to me